contracts/Raphael.sol

```diff
@@ -104,6 +104,10 @@ contract Raphael is ERC721Holder, Ownable, ReentrancyGuard {
         minVotesNeeded = 9644832 * 1e17; // 5% of initial distribution
     }
 
+    function getDidIVote(uint256 proposalIndex) public view returns (bool) {
+        return voted[proposalIndex][_msgSender()];
```

Comment:
if you decide to take an address `voter` as an argument, then this would become:
```
return voted[proposalIndex][voter];
```

Reply:
See the comment above.